index.html

```diff
@@ -36,5 +36,135 @@ <h3>LET'S CONNECT</h3>
+                 </ul>  
+            <button type="submit" class="project-button">See Project</button>
+         </div>
+</div>         
+<div class="grid-container4"> 
+           <div class="grid-item1"><img src="Snapshoot Portfolio(4).png" alt="portfolio"></div>
+                 <div class="grid-item2"><h2>Multi-Post <br>Stories</h2>
+            <ul>
+                <li style="list-style: none;" id="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately<br>
+                 personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                  <ul style="list-style: none;" class="language-button">
+                    <button type="submit"><li>html</li></button> 
+                  <button type="submit"><li>css</li></button>
+                    <button type="submit"><li>javascript</li></button>
+                 </ul>  
+            <button type="submit" class="project-button">See Project</button> 
+    </div>
+</div>    
+    <aside>
+        <h1>About <br>Myself</h1>
+    <p>I’m a software developer! I can help you <br>
+        build a product , feature or website Look <br>
+        through some of my work and <br>
+        experience! If you like what you see and <br>
+        have a project you need coded, don’t <br>hestiate to contact me.</p>
+          <h3>LET'S CONNECT</h3>
+        <ul style="list-style: none;" class="icon"> 
+            <a href="#"><li><img src="./Vector (1).png" alt="github"></li></a>
+            <a style="text-decoration: none;" href="#"><li>
+                <img src="./Vector (4).png" alt="lindedin">
+                <img src="./Vector (5).png" alt="">
+            </li></a>
+            <a href="#"><li><img src="./Vector (2).png" alt="victory"></li></a>
+            <a href="#"><li><img src="./Vector.png" alt="twitter"></li></a>
+            <a href="#"><li><img src="./Vector (3).png" alt="M"></li></a>
+        </ul>
+        <button type="submit" class="resume">Get my resume</button>
+            <ul  style="list-style: none;" class="language-container" >
+               <div class="super-class">
+                <li class="section-container">Languages</li>
+                <li><i class="fa fa-angle-down" aria-hidden="true"></i></li>
+            </div>
+                <ul style="list-style: none;" id="images">
+                  <li class="language"><img src="javascript.png" alt="javascript" >Javascript</li>
+                  
+                  <li class="language"><img src="html.png" alt="html">HTML</li>
+                  
+                  <li class="language"><img src="css.png" alt="css">CSS</li>
+                  
+                </ul>
+                <div class="super-class">
+                <li class="section-container">Frameworks</li>
+                <li><i class="fa fa-angle-right" aria-hidden="true"></i></li>
+            </div>
```

Comment:
- For semantic reasons and better accessibility, I'd suggest you always wrap `li` tags in a `ul` tag. Kindly fix this 👍 👍